### PR TITLE
feat(platform): Filter non-related PRs in "getPrList"

### LINKED ISSUE #7205: platform.getPrList should return only Renovate's PRs

### PR DESCRIPTION
**Pull request by zharinov**
Closes #7205

### CODE REVIEW

**Review by viceice — CHANGES_REQUESTED** [review]
need fixes

**Comment by zharinov** [comment]
🤷

**Comment by rarkins** [comment]
@zharinov @viceice maybe a node modules cache corruption? Same error on `master`

image

File: lib/platform/azure/index.ts [hunk 1]
```diff
@@ -104,6 +106,8 @@ export async function initRepo({
   repository,
   localDir,
   optimizeForDisabled,
+  branchPrefix,
```
Comment (rarkins) [inline lib/platform/azure/index.ts:109]: `branchPrefix` is configurable in the repo's config, so this won't work

**Review by rarkins — CHANGES_REQUESTED** [review]

**Comment by rarkins** [comment]
Back to requirements, see https://github.com/renovatebot/renovate/issues/7205#issuecomment-688900982

**Comment by zharinov** [comment]
Fixed on per-platform basis

**Pull request closed without merging**